client/src/dbus/mod.rs

```diff
@@ -37,6 +37,9 @@
 /// In general, you are supposed to use [`Service::default_collection`].
 pub const DEFAULT_COLLECTION: &str = "default";
 
+/// The default collection label.
+pub const DEFAULT_COLLECTION_LABEL: &str = "Default";
```

Comment:
Another open question I had, is there some sort of "default" label for the session collection?

Reply:
I don't think so, but I can dig later the code of gnome-keyring. This machine has a clean install and uses "Login" so maybe that is better.

Reply:
libsecret/libsecret/secret-methods.c:1058 uses 

```
		properties = _secret_collection_properties_new (_("Default keyring"));
		secret_service_create_collection_dbus_path (service, properties, "default",
```

whereas gnome-keyring/daemon/login/gdk-login.c uses
```
static GckObject*
create_login_keyring (GckSession *session, GckObject *cred, GError **error)
{
	GckBuilder builder = GCK_BUILDER_INIT;

	g_return_val_if_fail (GCK_IS_SESSION (session), NULL);
	g_return_val_if_fail (GCK_IS_OBJECT (cred), NULL);

	gck_builder_add_ulong (&builder, CKA_CLASS, CKO_G_COLLECTION);
	gck_builder_add_string (&builder, CKA_ID, "login");
	gck_builder_add_ulong (&builder, CKA_G_CREDENTIAL, gck_object_get_handle (cred));
	gck_builder_add_boolean (&builder, CKA_TOKEN, TRUE);

	/* TRANSLATORS: This is the display label for the login keyring */
	gck_builder_add_string (&builder, CKA_LABEL, _("Login"));

	return gck_session_create_object (session, gck_builder_end (&builder), NULL, error);
}
```